### PR TITLE
[BEAM-13977] [Website] Update Community landing page

### PR DESCRIPTION
**Pull request by bullet03**
[BEAM-13977]

- remove section with "send feedback"

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/#make-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by bullet03** [comment]
R: @AydarZaynutdinov

**Comment by AydarZaynutdinov** [comment]
@bullet03 Community/Landing page:
Join Beam Community -> Join the Beam Community

**Comment by bullet03** [comment]
> 

done

**Comment by dariabezkorovaina** [comment]
R: @pcoet

File: website/www/site/data/en/community_list.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-- title: Join Beam Community
+- title: Join the Beam Community
```
Comment (pcoet) [inline website/www/site/data/en/community_list.yaml:13]: "Beam community"

**Review by pcoet — APPROVED** [review]
One nitty suggestion - not blocking.

File: website/www/site/data/en/community_list.yaml (hunk 1 above)
Comment (bullet03) [reply website/www/site/data/en/community_list.yaml]: changed

**Review by AydarZaynutdinov — APPROVED** [review]
LGTM

**Pull request merged**